.github/workflows/publish-pr-review.yml

```diff
@@ -0,0 +1,28 @@
+name: Expo Review
+on: [pull_request]
+jobs:
+  publish:
+    name: Install and publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+          key: ${{ runner.os }}-${{ hashFiles('package.json', 'yarn.lock') }}
+      - run: yarn install --frozen-lockfile
+      - uses: expo/expo-github-action@v5
+        with:
+          expo-version: 4.x
+          expo-username: ${{ secrets.EXPO_CLI_USERNAME }}
+          expo-password: ${{ secrets.EXPO_CLI_PASSWORD }}
+      - run: expo publish --release-channel=pr-${{ github.event.number }}
```

Comment:
change we try to change `--release-channel=pr-${{ github.event.number }}` to `--release-channel=pr-${{ github.head_ref }}` ?

I think that will allow us to have a url of `https://expo.io/@sync-for-science/projects/discovery-mobile-ui?release-channel=pr-publish-pr-review` 

More about github context here:
https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context